pom.xml

```diff
@@ -170,11 +170,9 @@
       <version>3.0.0</version>
     </dependency>
     <dependency>
-      <groupId>com.gmail.filoghost.holographicdisplays</groupId>
+      <groupId>me.filoghost.holographicdisplays</groupId>
       <artifactId>holographicdisplays-api</artifactId>
-      <version>2.4.9</version>
-      <scope>provided</scope>
-      <optional>true</optional>
+      <version>3.0.0-20220815.173408-50</version>
```

Comment:
This is a SNAPSHOT version. This means that this:
```suggestion
      <version>3.0.0-SNAPSHOT</version>
```
always use the latest SNAPSHOT version.

Is there any specific reason you stick to this specific version instead of the Latest?

Reply:
When I started working on this PR, this was the latest version. I just didn't know that you could append SNAPSHOT to always have the latest version.